src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -586,12 +586,12 @@ protected virtual void ValidateAtHash(OpenIdConnectProtocolValidationContext val
         /// <summary>
         /// Validates that the <see cref="JwtSecurityToken"/> contains the nonce.
         /// </summary>
-        /// <param name="validationContext">a <see cref="OpenIdConnectProtocolValidationContext"/> that contains the 'nonce' to validate.</param>
-        /// <exception cref="ArgumentNullException">if 'idToken' is null.</exception>
-        /// <exception cref="ArgumentNullException">if 'validationContext' is null.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidNonceException">if <see cref="OpenIdConnectProtocolValidationContext.Nonce"/> is null and RequireNonce is true.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidNonceException">if the 'nonce' found in the 'id_token' does not match <see cref="OpenIdConnectProtocolValidationContext.Nonce"/>.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidNonceException">if <see cref="RequireTimeStampInNonce"/> is true and a timestamp is not: found, well formed, negatire or expired.</exception>
+        /// <param name="validationContext">A <see cref="OpenIdConnectProtocolValidationContext"/> that contains the 'nonce' to validate.</param>
+        /// <exception cref="ArgumentNullException">If 'idToken' is null.</exception>
+        /// <exception cref="ArgumentNullException">If 'validationContext' is null.</exception>
```

Comment:
change order: 'If validationContext ...' before 'if idToken...'
change text: 'If 'idToken' is null' to 'If validationContext.ValidatedIdToken is null'